catalog/transformer/catalog-transformer-xlsx/src/test/java/ddf/catalog/transformer/xlsx/XlsxMetacardUtilityTest.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transformer.xlsx;
+
+import ddf.catalog.data.BinaryContent;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.MetacardImpl;
+import java.util.Collections;
+import org.junit.Test;
+
+public class XlsxMetacardUtilityTest {
```

Comment:
Seems like we can make some tests that assert execution with an empty list of metacards as well as null attributes